.github/CODEOWNERS

```
@@ -7,7 +7,7 @@
 packages/datadog-ci/src/commands/coverage      @DataDog/datadog-ci-admins @DataDog/ci-app-libraries
 packages/datadog-ci/src/commands/deployment    @DataDog/datadog-ci-admins @DataDog/ci-app-libraries
 packages/datadog-ci/src/commands/dora          @DataDog/datadog-ci-admins @DataDog/ci-app-libraries
-packages/datadog-ci/src/commands/gate          @DataDog/datadog-ci-admins @DataDog/ci-app-libraries
+packages/plugin-gate                           @DataDog/datadog-ci-admins @DataDog/ci-app-libraries
```

Let's also own the file in base package (I'll update the How-to guide)
```suggestion
packages/base/src/commands/gate                @DataDog/datadog-ci-admins @DataDog/ci-app-libraries
packages/plugin-gate                           @DataDog/datadog-ci-admins @DataDog/ci-app-libraries
```